hw/top_earlgrey/cdc/cdc_waivers.tcl

```diff
@@ -11,13 +11,18 @@
 #   -comment {}
 # set_rule_status -rule { S_CONF_ENV } -status { Waived } -all_rule_data \
 #   -comment {}
+
+
 # Assumes modules defined in run-cdc.tcl
-if {[info exists $modules] == 0} {
-  puts "modules variable is not defined."
-} else {
-  foreach mod $modules {
-    if {[file exists "cdc_waivers.$mod.tcl"]} {
-      source "cdc_waivers.$mod.tcl"
-    }
+global modules
+global CDC_WAIVER_DIR
```

Comment:
is this required even if we `source` this script?

Reply:
Removed

Reply:
looks like these lines are still present? sorry if I missed something :)...

Reply:
Removed. I don't see those in the code..

Reply:
Oh I see. Sorry about that. I got confused since the first commit in this PR adds these lines and the second one removes them again.